client/lib/checkout/constants.js

```diff
@@ -22,6 +22,6 @@ export const PAYMENT_PROCESSOR_EBANX_COUNTRIES = {
 		],
 	},
 	MX: {
-		fields: [],
+		fields: [ 'phone-number', 'postal-code' ],
```

Comment:
Just a note as an explainer: `postal-code` is a required field for credit card payments anyway, but the reason why we must explicitly have it here is because we'll use the `< EbanxPaymentFields />` component in other places such as the redirect bank payment window, where no postal code field exists.